source/_integrations/alphaess.markdown

```diff
@@ -15,11 +15,15 @@ ha_codeowners:
 ---
 
 This integration pulls data from an [Alpha ESS](https://www.alpha-ess.com/) Energy Storage Solution.
+
+## Configuration
+
+The integration can be entirely setup from within the GUI.  Add the integration to Home Assistant and provide the username and password needed to logon to the Alpha ESS website.  Any Alpha ESS energy storage units registered with Alpha ESS will be automatically discovered and added to Home Assistant.  
```

Comment:
Please see how other integration pages do this, or check our developer documentation (which also provides information on documenting).

Use: `{% include includes/config_flow.md %}`

Reply:
Thanks - I have updated the documentation to follow this standard.